pkg/integration/replication/suite.go

```diff
@@ -214,17 +281,29 @@ func (suite *baseReplicationTestSuite) WaitForCommittedTx(
 	}
 }
 
-func (suite *baseReplicationTestSuite) SetupCluster(replicas int, syncFollowers int) {
-	suite.StartMaster(syncFollowers)
+func (suite *baseReplicationTestSuite) SetupCluster(syncReplicas, syncAcks, asyncReplicas int) {
```

Comment:
wdyt of adding syncReplicas and asyncReplicas in the baseReplicationTestSuite as params? Since we are calling SetupCluster always inside SetupSuite, it will make it simpler